package.json

```diff
@@ -43,9 +43,9 @@
     "@babel/runtime": "^7.10.5",
     "@tsconfig/node12": "^1.0.9",
     "agentkeepalive": "^3.5.2",
+    "buffer": "^6.0.3",
     "cbor-js": "^0.1.0",
     "cbor-sync": "^1.0.4",
-    "isomorphic-webcrypto": "^2.3.6",
```

Comment:
Are you sure this has no impact on browser builds?

Reply:
Yes, there will be no impact to our supported platforms. For very old browser versions and for internet explorer,  the change will impact support for file encryption. Customers who support those old browsers will be able to bring their own web cryptography polyfill to their applications in order to add the support since we will no longer bundle it.